src/api/libcellml/model.h

```diff
@@ -366,6 +366,89 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
      */
     bool fixVariableInterfaces();
 
+    /**
+     * @brief Add an import source item to this model.
+     *
+     * Add import source by reference to the model. If the import
+     * source item was previously in a different model, it is moved
+     * to this one, and the previous model's import source list is updated.
+     *
+     * The function will return @c false and no action is taken if:
+     *  - The @p imp pointer already exists in this model; or
+     *  - The @p imp is @c nullptr.
```

Comment:
Replace "imp" with "importSource".

Reply:
Same below.

Reply:
It has a @p for parameter, so needs to be the same as is listed in the declaration?

Reply:
... and elsewhere.

Reply:
Yes, I would expect the declaration to use `importSource` rather than `imp`, which in turn means that we should here use `@p importSource` rather than `@p imp`.

Reply:
It's not great to use variable names which shadow function names, which is why I chose to use `imp` and not `importSource`?

Reply:
?? The method is called `addImportSource()`, so where is the problem?

Reply:
If you declare a variable called `importSource` in the header, then you have a variable called `importSource` in the function.  Since we have a function called `importSource` in this class, then also having a variable with the same name isn't great?

Reply:
I have no problems with that, not least because this is exactly what is currently being done in `api/libcellml/importedentity.h` and in `api/libcellml/issue.h` where a method `importSource()` is defined and `importSource` is used as a parameter name for `setImportSource()`.

Reply:
Heh, I would rather change those ones too then ... I would have thought that there would be a clang flag for this kind of thing? Why do we only use `-Wshadow` in gcc? I'll change it to a longer variable name, but I really don't like shadowing others, so it won't be `importSource`.

Reply:
f85f512e